### DISCUSSION TITLE
does "--restart" CLI option combined with "update_rate" option deleting all elastic search documents regularly?

### DISCUSSION THREAD

**Discussion by JotaPin** [dadoonet/fscrawler#1677] [Q&A]
Dear all,
I have fscrawler launched with --restart, and update_rate: "5m" in the _settings.yaml.

* 88000 docs where seen in elasticsearch from docs.count in http://127.0.0.1:9200/_cat/indices?v
* now overnight they are only 44000

Does this indicate that fscawler restarts and erases all the docs in elasticsearch, each time it finishes plus 5 minutes?

Thanks in advance

**Comment by dadoonet** [comment]
No. The restart option will just reindex everything but will not delete any document.

**Reply by JotaPin** [reply]
Hi, thanks for your answer, in that case I don't see why the number of docs in elasticserch is decreasing to 44000, all the more that a file count on the disk gives 132000.... I am quite new to elasticsearch, this might be that there is something obvious I don't see...

**Reply by dadoonet** [reply]
I don't know. May be share all the exact steps you are doing? And share the fscrawler logs.
